protocol-designer/src/form-types.ts

```diff
@@ -364,9 +361,7 @@ export interface HydratedTemperatureFormData {
 }
 export interface HydratedHeaterShakerFormData {
   heaterShakerSetTimer: 'true' | 'false' | null
-  heaterShakerTimerMinutes: string | null
-  heaterShakerTimerSeconds: string | null
-  heaterShakerTimer?: string | null
+  heaterShakerTimer: string | null
```

Comment:
should this still have `undefined` type? since a timer is optional

```suggestion
  heaterShakerTimer?: string | null
```

Reply:
I think this field always exists and will be null if `setTimer` is false or null (similar to the legacy fields for `timerMinutes` and `timerSeconds`. I am happy to accept undefined if I am missing something though

Reply:
nah i think that makes sense! as long as the default value for it is null